File: primeN.js
let number = 8;
let counter = 0;
for(i= 1; i < number; i++){
  if(number/i == Math.trunc(number/i)){
    counter++;
  }
}
if(counter > 2 || number == 1){
  console.log('not prime');
}
else{
  console.log('prime');
}
//if a number can be divided by more than 2 numbers (for prime --> 1 and the number itself), it's not a prime number.
//1 can only be divided by one number, 1 itself, so with this definition 1 is not a prime number.

File: primeN.py
import math
n = 4
#index is the count of dividers the number's got
index = 0
for counter in range(1, n+1):
    if n/counter == math.trunc(n/counter):
        index = index + 1
if index > 2 or n == 1:
    print('not a prime number')
else:
    print('prime numebr')
